app/models/concerns/thredded/post_common.rb

```diff
@@ -54,9 +54,31 @@ def context_options
       {
         asset_root: Rails.application.config.action_controller.asset_host || '',
         post:       self,
+        whitelist:  sanitize_whitelist,
       }
     end
 
+    def sanitize_whitelist
+      HTML::Pipeline::SanitizationFilter::WHITELIST[:transformers] += [
+        lambda do |env|
+          node = env[:node]
+
+          a_tags = node.css('a')
```

Comment:
This would definitely be easier! However, these attributes would be applied to internal links too. 

External links? Ok, I'm totally on board :). Relative path links, though? Eeeennnhhh notsomuch.

That's why I went the route of the transformer thing ... ask the link `href` where it's going, then add the attributes.

Whataya say?!